### PR TITLE
chore: use enums and css helper 

### PR DESCRIPTION
**Pull request by IlyaBondar**
### Description
- use enums/constants
- use CSS-helper `getByHighlightColor`
- fix dependencies

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

File: .eslintrc.json [hunk 1]
```diff
@@ -26,7 +26,10 @@
       { "argsIgnorePattern": "^_" }
     ],
     "no-restricted-imports": ["error", { "paths": ["react-i18next"] }],
-    "tailwindcss/no-custom-classname": "error",
+    "tailwindcss/no-custom-classname": ["error", { "callees": [
```
Comment (mikitabut) [inline .eslintrc.json]: This is duplication of line 16 - use it there because it will affect not only 'no-custom-classname' but other rules also

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

File: src/types/openai.ts [hunk 2]
```diff
@@ -1,7 +1,9 @@
-export type OpenAIEntityModelType = 'model';
-export type OpenAIEntityApplicationType = 'application';
-export type OpenAIEntityAssistantType = 'assistant';
-export type OpenAIEntityAddonType = 'addon';
+import { EntityType } from './common';
+
+export type OpenAIEntityModelType = EntityType.Model;
```
Comment (mikitabut) [inline src/types/openai.ts]: Do we need this if you already have enum?

File: .eslintrc.json (hunk 1 above)
Comment (IlyaBondar) [reply .eslintrc.json]: Hm, you are right we need to apply all rules(order, no-custom and etc.) except `tailwindcss/no-contradicting-classname`(not so smart that we are choosing between) for `getByHighlightColor`

File: e2e/src/testData/conversationHistory/conversationData.ts [hunk 3]
```diff
@@ -1,7 +1,9 @@
 import { DateUtil } from '@/e2e/src/utils/dateUtil';
 import { GeneratorUtil } from '@/e2e/src/utils/generatorUtil';
 
-import { Conversation, Message, Stage } from '@/src/types/chat';
+
```
Comment (mikitabut) [inline e2e/src/testData/conversationHistory/conversationData.ts]: as for me too much empty rows :)

**Review by mikitabut — DISMISSED** [review]

**Review by mikitabut — APPROVED** [review]

**Pull request merged**